consensus/istanbul/validator/default.go

```diff
@@ -395,7 +396,7 @@ func (valSet *defaultSet) F() int {
 
 func (valSet *defaultSet) Policy() istanbul.ProposerPolicy { return valSet.policy }
```

Comment:
Missing description. How about adding the description in the next PR?

Reply:
Sure. I'll make a new pr for adding descriptions.